polygeist/tools/cgeist/Test/Verification/sycl/kernels.cpp

```diff
@@ -8,6 +8,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+// clang-format off
```

Comment:
`llvm/polygeist/tools/cgeist/Test/.clang-format` has `DisableFormat: true`. Why this change is needed?